### PR TITLE
updates from the flutter gallery rename

### PR DESCRIPTION
**Pull request by devoncarew**
Minor updates from the flutter gallery rename. @HansMuller

### CODE REVIEW

File: examples/flutter_gallery/android/AndroidManifest.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
      Use of this source code is governed by a BSD-style license that can be
      found in the LICENSE file.
  -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.flutter.MaterialGallery" android:versionCode="1" android:versionName="0.0.1">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.flutter.FlutterGallery" android:versionCode="1" android:versionName="0.0.1">
```
Comment (eseidelGoogle) [inline examples/flutter_gallery/android/AndroidManifest.xml]: I believe java is case sensitive here, but maybe we should take this opportunity to make the ios and android ones match?

Comment (eseidelGoogle) [inline examples/flutter_gallery/android/AndroidManifest.xml]: And/or possibly jsut name it io.flutter.gallery :D

**Comment by HansMuller** [comment]
Thanks for catching those, LGTM.

File: examples/flutter_gallery/android/AndroidManifest.xml (hunk 1 above)
Comment (devoncarew) [inline examples/flutter_gallery/android/AndroidManifest.xml]: renamed to `io.flutter.gallery`

**Pull request merged**